### PR TITLE
chore(vue): Update vue-slider-component 

### PR DESCRIPTION
**Pull request by mohdashraf010897**
**PR Type** `Bug Fix`

**Description** 
- https://github.com/appbaseio/reactivesearch/issues/1815
   - updated `vue-slider-component` to **v3.2.15**.
   - Demo 👉 https://www.loom.com/share/f436c7c916484b53a21fdfc273138f41
   - **CSB(s)**
      - [RangeSlider](https://codesandbox.io/embed/heuristic-browser-6lhnr?fontsize=14&hidenavigation=1&theme=dark)
      - [DynamicRangeSlider](https://codesandbox.io/embed/zealous-einstein-kgfuq?fontsize=14&hidenavigation=1&theme=dark)

[📔  Notion](https://www.notion.so/appbase/ReactiveSearch-issues-86750916b182477793caefc4d89b377f)

### CODE REVIEW

File: packages/web/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@appbaseio/reactivesearch",
-	"version": "3.26.1",
+	"version": "3.26.1-alpha.2",
```
Comment (bietkul) [inline packages/web/package.json:3]:
```suggestion
	"version": "3.26.1",
```

File: packages/vue/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@appbaseio/reactivesearch-vue",
-	"version": "1.24.3",
+	"version": "1.24.3-alpha.1",
```
Comment (bietkul) [inline packages/vue/package.json:3]:
```suggestion
	"version": "1.24.3",
```

**Review by bietkul — CHANGES_REQUESTED** [review]
@mohdashraf010897 This PR has the following issues:
- Too many API calls on dragging the slider. It is happening in `DRS`.
- RangeSlider is not working with `RangeInput`. Clear the `SelecteFilter` and it wouldn’t update the `RangeInput` component.
- ReactiveSearch CSS doesn't seem to work. Just compare it with the stable version.
- Test it with SSR too

I would recommend you to follow the migration guide for `vue-slider` to upgrade the version.

Create a separate PR for `renderItem` issue in list components. Check for `Vue` as well.

**Comment by mohdashraf010897** [comment]
fixed here https://github.com/appbaseio/reactivesearch/pull/1853

**Pull request closed without merging**